tests/mock_network/churn.rs

```diff
@@ -560,3 +560,33 @@ fn messages_during_churn() {
         verify_invariant_for_all_nodes(&network, &mut nodes);
     }
 }
+
+#[test]
+fn remove_unresponsive_node() {
+    let min_section_size = 4;
+    let network = Network::new(min_section_size, None);
+    let mut nodes = create_connected_nodes(&network, 2 * min_section_size);
+    poll_and_resend(&mut nodes);
+    // Pause a node to act as non-responsive.
+    let mut rng = network.new_rng();
+    let index = rng.gen_range(0, nodes.len());
+    let _paused_node = nodes[index].name();
+    let _ = unwrap!(nodes.remove(index).inner.pause());
+
+    // Sending some user events to create a sequence of observations.
+    for _ in 0..2 * min_section_size {
```

Comment:
I do not understand that count, is it related to threshold

Reply:
yes. mainly just the number of nodes here.

Reply:
yes. mainly just the number of nodes here.